opennms-container/minion/container-fs/confd/templates/prom-jmx-exporter.yaml.tmpl

```diff
@@ -0,0 +1,25 @@
+#
+# DON'T EDIT THIS FILE :: GENERATED WITH CONFD
+#
+{{$jmxExporterPath := "/java/agent/prom-jmx-exporter" -}}
```

Comment:
Would it be better to simply not generate the file in the case they haven't configured the exporter rather than have default values here?

Reply:
Yes, agree. I'll look into this. 👍